src/functions/postResult/application/save-result-service.ts

```diff
@@ -1,5 +1,48 @@
 import { StandardCarTestCATBSchema } from '@dvsa/mes-test-schema/categories/B';
+import * as mysql from 'mysql2';
+import { IntegrationType } from '../domain/result-integration';
+import { getConnection } from '../../../common/framework/mysql/database';
+import { buildTestResultInsert, buildUploadQueueInsert } from '../framework/database/query-builder';
 
-export const saveTestResult = (testResult: StandardCarTestCATBSchema): Promise<void> => {
-  return Promise.resolve();
+export const saveTestResult = async (testResult: StandardCarTestCATBSchema): Promise<void> => {
+  const connection: mysql.Connection = getConnection();
```

Comment:
Should we close this connection at some point?   
Are we planning to introduce the connection pooling in this lambda (similar to the pooler one) or we're happy to just manually open/close connections?

Reply:
Added closing of connections.

Reply:
Don't see a reason for a connection pool in this Lambda, as everything that we're saving needs to be bounded by the same transaction - multiple connections wouldn't help us here.